private/model/api/operation.go

```diff
@@ -70,6 +70,7 @@ var tplOperation = template.Must(template.New("operation").Funcs(template.FuncMa
 	"GetCrosslinkURL": GetCrosslinkURL,
 }).Parse(`
 {{ $reqType := printf "%sRequest" .ExportedName -}}
+{{ $pagerType := printf "%sPager" .ExportedName -}}
```

Comment:
We want to make sure this `__Pager` type won't collide with modeled shapes.

Reply:
Ah! Good point. Will fix.